### PR TITLE
Put api documentation under the same path prefix

### PR DESCRIPTION
**Pull request by Benjaminimal**
Since the swagger UI and the OpenAPI specification are both part of the API documentation I put the former under `/api-doc/` as well.
Also mentioned the latter in the backend readme.

## Basics

- [x] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [x] The buildserver is happy.
- [x] The PR is rebased with current master.

## Checklist

- [ ] I added unit tests for my code
- [x] I fully described what my PR does in the documentation
      (not in the PR description)
- [x] I fixed all affected documentation
- [ ] I fixed all affected decisions
- [ ] I added code comments, logging, and assertions as appropriate
- [ ] I mentioned [every code](/.reuse/dep5) not directly written by me in [reuse syntax](https://reuse.software/)

## Review

- [ ] Documentation is conforming to [our Documentation Guidelines](/doc/documentation.md)
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to our Coding Guidelines
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)

### CODE REVIEW

**Comment by kitzbergerg** [comment]
@lukashartl You need to change the proxy again after this is merged (remove the /swagger-ui/ path)

**Review by kitzbergerg — APPROVED** [review]

**Comment by lukashartl** [comment]
> @lukashartl You need to change the proxy again after this is merged (remove the /swagger-ui/ path)

yeah, I should probably stop fixings thing right in the meeting and should always wait for issues 🙈 
But even if we keep the proxy-route, it wouldn't be a problem 🙉

**Review by lukashartl — APPROVED** [review]

File: backend/src/config/api_doc.rs [hunk 1]
```diff
@@ -47,5 +47,5 @@ pub fn config(cfg: &mut web::ServiceConfig) {
     let mut openapi = SeedApiDoc::openapi();
     openapi.merge(PlantsApiDoc::openapi());
 
-    cfg.service(SwaggerUi::new("/swagger-ui/{_:.*}").url("/api-doc/openapi.json", openapi));
+    cfg.service(SwaggerUi::new("/api-doc/swagger-ui/{_:.*}").url("/api-doc/openapi.json", openapi));
```
[the remark this answers is not in the document]
Comment (Benjaminimal) [reply backend/src/config/api_doc.rs]: Sure, sounds good.
https://github.com/ElektraInitiative/PermaplanT/pull/174/commits/6a8aad682ea2b0e8954654e7aef50e86e6818ce8

**Comment by markus2330** [comment]
Great job! :heart:

**Comment by Benjaminimal** [comment]
@lukashartl
the paths are now

- `/doc/api/swagger/ui/`
- `/doc/api/openapi.json`

please adjust the proxy accodingly

**Comment by lukashartl** [comment]
> @lukashartl the paths are now
> 
>     * `/doc/api/swagger/ui/`
> 
>     * `/doc/api/openapi.json`
> 
> 
> please adjust the proxy accordingly

Done

**Pull request merged**